sunbird-platform-core/common-util/src/main/java/org/sunbird/common/util/Matcher.java

```diff
@@ -0,0 +1,21 @@
+package org.sunbird.common.util;
+
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * this class is used to match the identifiers.
+ *
+ */
+public class Matcher {
+
+    /**
+     * this method will match the two arguments , equal or not
+     * @param i1
+     * @param i2
+     * @return boolean
+     */
+    public static boolean matchIdentifiers(String i1,String i2){
```

Comment:
Don't used method param as i1 and i2 .
u can use  firstVal , secondVal or value1 , value2

Reply:
just add in doc if both string value are null or empty then also it will return true.